docs/persistence/kafka/README.md

```diff
@@ -1,11 +1,6 @@
 # Kafka
 
-!!! warning
-    This feature applies only to _Aiven hosted Kafka_. On-premises Kafka is deprecated, and creating new topics on-premises was disabled summer 2021. For on-premises Kafka, see [on-premises Kafka documentation](https://confluence.adeo.no/display/AURA/Kafka).
-
-## Abstract
-
-NAV uses Aiven hosted Kafka. Access to Kafka is granted by defining a `Topic` resource in one of our Kubernetes clusters.
+NAV only uses Aiven hosted Kafka, the on-premis version of Kafka was disabled in 2021. Access to Kafka is granted by defining a `Topic` resource in one of our Kubernetes clusters.
```

Comment:
```suggestion
NAV uses Aiven hosted Kafka, the on-premise version of Kafka has been deprecated since 2021. Start using Kafka by creating a `Topic` resource in one of our Kubernetes clusters.
```